### ISSUE TITLE
osx: build fails with current macports:  undeclared identifier 'gdk_quartz_window_get_nswindow'

### ISSUE THREAD

**Issue by MStraeten** [darktable-org/darktable#4338]
compiling currentmaster resukts in:

[ 31%] Building CXX object src/CMakeFiles/lib_darktable.dir/osx/osx.mm.o
.../darktable/src/osx/osx.mm:69:24: error: use of undeclared identifier
      'gdk_quartz_window_get_nswindow'
    NSWindow *native = gdk_quartz_window_get_nswindow(window);

**Platform (please complete the following information):**
 - Darktable Version: current master
 - OS: OSX Catalina
 - just updated macports

**Comment by MStraeten** [comment]
comparing macports with last timemachine results in:
/opt/local/include/gtk-3.0/gdk/quartz/gdkquartzwindow.h changed,
```
GDK_AVAILABLE_IN_ALL
NSWindow *gdk_quartz_window_get_nswindow (GdkWindow *window);
GDK_AVAILABLE_IN_ALL
NSView   *gdk_quartz_window_get_nsview   (GdkWindow *window);
```
was removed

**Comment by parafin** [comment]
Which GTK version is that?

**Comment by MStraeten** [comment]
gtk3 3.24.14+quartz
old was: gtk3 3.24.12+quartz

**Comment by parafin** [comment]
OK, I will check, there is some mention of quartz fullscreen fixes in GTK 3.24.14, so maybe we don't require that call anymore, but changing API in minor release is bad even for GTK guys.

**Comment by parafin** [comment]
fixed in master and 3.0.x
